Makefile

```diff
@@ -0,0 +1,42 @@
+# This Makefile is meant to be used by people that do not usually work
+# with Go source code. If you know what GOPATH is then you probably
+# don't need to bother with make.
+
+GOBIN = $(shell pwd)/build/bin
```

Comment:
We actually have a `build` folder in the Swarm codebase that has source code in it, unlike many projects that use it for artifacts and binaries. So adding a `bin` folder in it is kinda weird.

Reply:
again, this is the way it has always been. I am not changing anything at all from previous behaviour.